lms/djangoapps/lms_xblock/runtime.py

```diff
@@ -157,7 +157,7 @@ def __init__(self, **kwargs):
             services['badging'] = BadgingService(course_id=kwargs.get('course_id'), modulestore=store)
         self.request_token = kwargs.pop('request_token', None)
         services['teams'] = TeamsService()
-        services['teams_config'] = TeamsConfigurationService()
+        services['teams_configuration'] = TeamsConfigurationService()
```

Comment:
Should we leave both here for now, and after your ORA PR gets merged/releases, get rid of the `teams_config` key?  Then you don't have to worry about the timing of releases or checking out of various branches, etc.

Reply:
well, we don't use it anywhere currently, my other PR is adding in the usage of it in this context, so I don't think anything will break

Reply:
Oh, you're right, I keep confusing this with the service we hook into the CMS runtime.